gravitee-am-repository/gravitee-am-repository-mongodb/src/main/java/io/gravitee/am/repository/mongodb/management/internal/model/ApplicationSAMLSettingsMongo.java

```diff
@@ -25,6 +25,8 @@ public class ApplicationSAMLSettingsMongo {
     private String attributeConsumeServiceUrl;
     private String singleLogoutServiceUrl;
     private String certificate;
+    private boolean wantResponseSigned;
```

Comment:
Should also defne the default value here  link https://github.com/gravitee-io/gravitee-access-management/pull/2197/files#diff-70a5fcd7cf0bbcedd548b7f8bd379a7768991e8b15b9d0032065bdac139de7f1R47 otherwise existing app will have the flag disabled.